### PR TITLE
UGACT-602 Navigation style adjustments

### PR DESCRIPTION
**Pull request by KamilChoroba**
* Style adjustments for the smyle header integration
* Allow max width for submenu-items
* Break submenu-items in case there is
  not enough space to visualize the full text
* Add css class for smyle icon

### CODE REVIEW

**Review by ctoutountzidis — APPROVED** [review]

File: src/06-components/organisms/navigation-v2/navigation-v2.scss [hunk 1]
```diff
@@ -28,3 +28,30 @@ $navigation-breakpoint: 1000px;
...
+  }
+
+  .title {
+    &::after {
+      margin-top: 0;
+    }
+  }
+}
+
+.sc-navigation-v2 button.smyle-header-menu:after {
```
Comment (lakhaNamdhari) [inline src/06-components/organisms/navigation-v2/navigation-v2.scss:49]: This block can already be nested under `button.smyle-header-menu {`

File: src/06-components/organisms/navigation-v2/navigation-v2.scss [hunk 2]
```diff
@@ -28,3 +28,30 @@ $navigation-breakpoint: 1000px;
     z-index: 140; // more than header
   }
 }
+
+.sc-navigation-v2 button.smyle-header-menu {
```
Comment (lakhaNamdhari) [inline src/06-components/organisms/navigation-v2/navigation-v2.scss:32]: This block can already be nested under `.sc-navigation-v2` at line:6

File: src/06-components/organisms/navigation-v2/navigation-v2.scss [hunk 3]
```diff
@@ -28,3 +28,30 @@ $navigation-breakpoint: 1000px;
...
+
+.sc-navigation-v2 button.smyle-header-menu:after {
+  margin-top: 0;
+}
+
+.smyle-header-icon {
+  width: 55px;
+  height: 33px;
+  margin-right: 8px;
+}
```
Comment (lakhaNamdhari) [inline src/06-components/organisms/navigation-v2/navigation-v2.scss:57]: Could you please add this block under scope of `..sc-navigation-v2`?  defined (line:6)

File: src/06-components/organisms/navigation-v2/navigation-v2-mixin.scss [hunk 4]
```diff
@@ -367,6 +368,7 @@ $header-height: 71px;
       a {
         padding-left: 0;
         padding-right: 0;
+        white-space: nowrap;
```
Comment (lakhaNamdhari) [inline src/06-components/organisms/navigation-v2/navigation-v2-mixin.scss:371]: We should check the impact of these style changes on dealer navigation as it seems to use this mixin.

https://github.com/AutoScout24/as24-dealer-navigation/blob/2dd9c685205d9513b9f786b1f9e635d33756e162/ui/scss/header.scss#L15

you could quickly check with Martin Lechner as they own dealer navigation AFAIK.

**Review by lakhaNamdhari — CHANGES_REQUESTED** [review]
Hey Kamil - I have added few comments.

File: src/06-components/organisms/navigation-v2/navigation-v2-mixin.scss (hunk 4 above)
Comment (KamilChoroba) [reply src/06-components/organisms/navigation-v2/navigation-v2-mixin.scss:371]: Seems to look fine. I've tested the dealer-navigation locally and it looks good. I think they use somehow another prefix `as24dn__nav`.

**Review by lakhaNamdhari — APPROVED** [review]

**Pull request merged**